packages/wallets/src/components/WalletAddedSuccess/WalletAddedSuccess.tsx

```diff
@@ -43,6 +43,7 @@ const WalletAddedSuccess: React.FC<TWalletAddedSuccessProps> = ({
                 balance={displayBalance}
                 currency={currency || 'USD'}
                 landingCompanyName={landingCompany}
+                padding='1.6rem'
```

Comment:
as per other comments in other PRs - this belongs to .css
I would love to see wallets being fully responsible like it should be - inline styles, among other problems, will make it extremely hard to go into that direction

https://github.com/binary-com/deriv-app/pull/11407/files#r1394266831